frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -252,11 +260,19 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
     selectedTargetNamespace,
   );
 
+  const tokenType = 'role ARN';
+  const tokenizedAuth = getQueryArgument('tokenizedAuth');
+  const [showWarn, setShowWarn] = React.useState(true);
```

Comment:
Would be great if this hook could live closer to the beginning of the component per best practices.  And the name is a bit generic.  Maybe `showSTSWarn`?